.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1447,6 +1447,28 @@ Assist test plan is in the core section instead of WebUI as most functionality i
   - [ ] A Bash command can be generated in the above window.
   - [ ] When an output is selected in the Terminal "Explain" option is available, and it generates the summary.
 
+## IGS:
+- [ ] Access Monitoring
+  - [ ] Verify that users can run custom audit queries.
+  - [ ] Verify that the Privileged Access Report is generated and periodically refreshed.
+
+- [ ] Access List
+  - [ ] Verify Access List membership/ownership/expiration date.
```

Comment:
Let's add test case scenarios like owners can only add/remove members and not change other properties, and other modification use-cases.